tests/integration/telemetry/stackdriver/testdata/server_audit_log.json.tmpl

```diff
@@ -0,0 +1,24 @@
+{
+    "httpRequest": {
+      "requestMethod": "GET",
+      "requestUrl": "http://srv.{{ .EchoNamespace }}.svc.cluster.local/audit-all",
```

Comment:
I think you should also check that `/audit-none` is not audit logged, and `/foo` and `/bar` is logged when header matched.